### ISSUE TITLE
Bug: Horizontal line sometimes visible at the centre of the Donut chart

### ISSUE THREAD

**Issue by deandwill** [carbon-design-system/carbon-charts#512]
#### I am submitting a...

- [ ] Feature request
- [ ] Design defect
- [x] Source code defect
- [ ] Demo/documentation defect
- [ ] Other

#### charts version:

<script src="https://unpkg.com/@carbon/charts/bundle.js"></script>
<link rel="stylesheet" href="https://unpkg.com/@carbon/charts/styles-g100.css" />

### Issue description
... when it should be above the Donut chart ( associated with the last percentage )

#### Steps to produce the issue

eg see http://peretz1.fyre.ibm.com:3040/edt/   ... and click on "Activity" in he left nav.

### Screenshot or recording
<img width="678" alt="Screen Shot 2020-02-21 at 12 05 05 PM" src="https://user-images.githubusercontent.com/446838/75055473-11027e80-54a3-11ea-8b4e-3975ad5bcd9b.png">

**Comment by deandwill** [comment]
Working now.  Thanks !
<img width="725" alt="Screen Shot 2020-03-11 at 12 02 41 PM" src="https://user-images.githubusercontent.com/446838/76437684-4e667780-6390-11ea-8928-abc38a382253.png">
